dotcom-rendering/.eslintrc.js

```diff
@@ -120,7 +120,8 @@ module.exports = {
 		'no-empty-pattern': 'error',
 		'no-fallthrough': 'off', // We use 'noFallthroughCasesInSwitch' in tsconfig.json as this respects types
 		'no-param-reassign': 'error',
-		'no-shadow': 'warn',
+		'no-shadow': 'off', // We use the typescript-eslint version as eslint false positives on enums
+		'@typescript-eslint/no-shadow': ['error'],
```

Comment:
Without making this change, we'd getting a `variable is already declared in upper scope` warning from ESLint, despite that not being the case. Using the typescript version of this rule fixes this problem

Reply:
Was this needed even without the use of TypeScript `enum`s?